lib/core/src/examples/text_selecting.rs

```diff
@@ -59,7 +61,19 @@ pub fn run_example_text_selecting() {
         world.add_child(&text_field);
         text_field.update();
 
+        let text_field_rc       = Rc::new(RefCell::new(text_field));
+        let text_field_on_click = text_field_rc.clone_rc();
```

Comment:
`clone_rc` is deprecated (my failure its still in prelude). Can I ask you to use `clone_ref` instead?